DNN Platform/Library/Services/Social/Messaging/Data/DataService.cs

```diff
@@ -116,7 +116,7 @@ public int CreateMessageReply(int conversationId, int portalId, string body, int
         /// <returns>The count of recipients</returns>
         public int CheckReplyHasRecipients(int conversationId, int userId)
         {
-            return userId <= 0 ? 0 : _provider.ExecuteScalar<int>("CoreMessaging_CheckReplyHasRecipients", conversationId, userId);

```

Comment:
these changes seems commit in 6a374e8f for performance stuff, should it really need remove?